### PR TITLE
Create the CurrencyBalance interface.

### PR DESCRIPTION
**Pull request by skubarenko**
Change the getBalance method of the ExchangeService interface.
The first part of the balance implementation which is described in #67

### CODE REVIEW

File: src/core/balance.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export interface CurrencyBalance {
```
Comment (maxima-net) [inline src/core/balance.ts]: Why don't rename this file? I suggest 'CurrencyBalance.ts' as a name.

**Review by maxima-net — CHANGES_REQUESTED** [review]

File: src/core/balance.ts (hunk 1 above)
Comment (skubarenko) [reply src/core/balance.ts]: 'currency-balance.ts' is a more consistent variant. Ok, I'll rename it

Comment (maxima-net) [reply src/core/balance.ts]: Of course it is. I’ve done a mistake. Sorry.

File: src/core/currency-balance.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+export interface CurrencyBalance {
+    readonly amount: number;
```
Comment (maxima-net) [inline src/core/currency-balance.ts]: We've also decided that the `amount` property should be `allAmount`. Isn't it?

**Review by maxima-net — CHANGES_REQUESTED** [review]

File: src/core/currency-balance.ts (hunk 2 above)
Comment (skubarenko) [reply src/core/currency-balance.ts]: Yeah, right

**Review by maxima-net — APPROVED** [review]

**Pull request merged**